assets/stylesheets/sass/main/main.scss

```diff
@@ -378,3 +393,45 @@ figure {
     width: 100%;
   }
 }
+
+.website-links {
+  border-bottom: 1px solid $light-gray;
+  color: $white;
+  list-style: none;
+  margin-bottom: 1em;
+  padding: 0 0 1em 0;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `0 0 1em` instead of `0 0 1em 0`